### ISSUE TITLE
Matas feedback -  Mute/unmute audio

### ISSUE THREAD

**Issue by JesperBerthelsen** [bambuser/bambuser-livevideoshoppingplayer-sdk-ios#9]
We are unable to find anything related to mute/unmute audio which we need in “small” mode.

**Comment by JesperBerthelsen** [comment]
Your comment:
Oh bummer 😞 unfortunately there is no interface to explicit toggle mute on/off. But, that should be pretty straight forward to implement. We will add a task for that.

**Comment by ken-bam** [comment]
We have added a `volume` property to the `LiveVideoShoppingPlayerView` that take values in the range 0.0 - 1.0. To mute set it to 0.0 and to unmute > 0.0. Note! When muting hide the player muteOverlay by calling:

```
self.playerView.invoke(.hideUI(sections: PlayerUISections(unmuteOverlay: .hidden)), completion: {_ in
    // Do stuff
})
```

**Comment by ken-bam** [comment]
I close this issue for now due to lack of feedback. The possibility to change volume and thereby muting the player has been available since version 0.8.0.
